query/tests/it/interpreters/interpreter_interceptor.rs

```diff
@@ -66,8 +66,8 @@ async fn test_interpreter_interceptor() -> Result<()> {
             "+----------+--------------+-----------+--------------+-----------+------------+-----------------+--------------+---------------+-------------+--------------+------------+------------------------------------------------------+----------+---------------------------------------------------------------------------+",
             "| log_type | handler_type | cpu_usage | memory_usage | scan_rows | scan_bytes | scan_partitions | written_rows | written_bytes | result_rows | result_bytes | query_kind | query_text                                           | sql_user | sql_user_privileges                                                       |",
```

Comment:
The main branch has remove the column `memory_usage` :D

Reply:
got it, 3Q!

Reply:
Merged with the main branch, `memory_usage` is still there ...

Reply:
Sorry, I have fixed it in this  PR: https://github.com/datafuselabs/databend/pull/3809